src/test/java/RandomizerTest.java

```diff
@@ -0,0 +1,16 @@
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertFalse;
+
+public class RandomizerTest {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [All classes and interfaces must belong to a named package](https://www.codacy.com/app/mollie-c-stephenson/TicTacToe/file/2534705908/issues/source?bid=3284885&fileBranchId=3284887#l8)